core/src/main/java/hudson/model/InvisibleAction.java

```diff
@@ -24,12 +24,16 @@
 package hudson.model;
 
 /**
- * Partial {@link Action} implementation that doesn't have any UI presence.
+ * Partial {@link Action} implementation that doesn't have any UI presence (unless the {@link #getUrlName()} is overrided).
```

Comment:
This seems slightly confusing as just having a URL does not make for a UI presence, at least not in the most direct sense of having a presence on a side panel.

Reply:
In a way I'm contradicting what @oleg-nenashev wrote in his older comment. It really depends on what "UI presence" means. Perhaps just clarify that there can be a URL space with views etc., but there cannot be a presence in a sidepanel?

I guess unless you override `action.jelly` or what it's called. Well…

Let's just merge as is, any second spent on this is wasted.